src/main/scala/uk/gov/hmrc/nonrep/attachment/server/ServiceConfig.scala

```
@@ -1,17 +1,22 @@
 package uk.gov.hmrc.nonrep.attachment
 package server
 
-import java.net.URI
-
 import com.typesafe.config.{Config, ConfigFactory}
 
-import scala.jdk.CollectionConverters._
+import java.net.URI
 
 class ServiceConfig(val servicePort: Int = 8000) {
 
   val appName = "attachment-processor"
-  val port: Int = sys.env.get("REST_PORT").map(_.toInt).getOrElse(servicePort)
-  val env: String = sys.env.get("ENV").getOrElse("local")
+  val port: Int = sys.env.get("REST_PORT").fold(servicePort)(_.toInt)
+  val env: String = sys.env.getOrElse("ENV", "local")
+  val glacierNotificationsSnsTopicArn: String = if (env == "local") "local" else glacierSNSSystemProperty
```

Allows the service to run locally without `GLACIER_SNS` being set but mandates it in deployed environments.

I thought better to fail fast than allow it to fail at the start of New Years day.